### DISCUSSION TITLE
How to contribute ?

### DISCUSSION THREAD

**Discussion by Z35hLy48k** [TriliumNext/Trilium#2512] [General]
This is one of the best note taking apps out there that fits my needs. I would love to contribute to the project.
Is there somewhere I can begin with, or a roadmap, or a wiki for developers/contributors to understand the code base ?

If not, I think this project can really benefit from a contribution doc/wiki to help people contribute :).

**Comment by zadam** [comment]
Hi, there isn't any specific guide. You can try to clone the repo and play with it.

You can check the issue tracker for some available issues, although not everything is actually planned / desirable.

This project is largely a one man show and I prioritize coding to other activities (like grooming the issues).

**Reply by Z35hLy48k** [reply]
Yes I realized this project is kind of a one man show, and I think it could benefit from people's contribution. 
A great way would be to have a proper Contribution guide to start with, don't you think ?

**Reply by zadam** [reply]
That alone would be a major effort and is no guarantee contributors would come. It's also a moving target.

That doesn't mean I don't want to do it, but ATM I have higher priority things to work on.
